sonic_data_client/virtual_db.go

```diff
@@ -419,7 +468,7 @@ func v2rEthPortPfcwdStats(paths []string) ([]tablePath, error) {
 // Populate real data paths from paths like
 // [COUNTER_DB COUNTERS Ethernet* Queues] or [COUNTER_DB COUNTERS Ethernet68 Queues]
 func v2rEthPortQueStats(paths []string) ([]tablePath, error) {
-	separator, _ := GetTableKeySeparator(paths[DbIdx])
+	separator, _ := GetTableKeySeparator(paths[DbIdx], "")
```

Comment:
if the blank string is intended to get separator for default namespace, it's better to use constant for default database `sonic_db_config.db_config.SONIC_DEFAULT_NAMESPACE` or `GetDbDefaultNamespace()` from the same package

Reply:
even though value is same the SONIC_DEFAULT_NAMESPACE  is used for some other purpose and has different meaning. To replace constant we will need to use other macro. This can be enhanced as part of other pr regarding cleanup the code